source/initial-dns-seedlist-discovery/initial-dns-seedlist-discovery.rst

```diff
@@ -83,13 +86,53 @@ the SRV record. For example, when using the non-FQDN ``servername`` and the
 default domain set to ``example.com``, a Client should end up requesting the
 SRV record for ``servername.example.com``.
 
+Default Connection String Options
+---------------------------------
+
+A Client MUST also query the DNS server for TXT records. If available, these
+TXT records provide default connection string options. In most cases only one
+TXT record is necessary, unless the default options exceed the maximum length
+of 255 characters that TXT records allow for. To allow for this case, a Client
+MUST allow for multiple TXT records for the same host name and consider the
+options in all of them.
+
+Information returned with each TXT record is a simple URI string, just like
+the options in a connection string.
+
+Although options in a connection string are separated by a ``&``, this
+character MUST NOT be present to separate options between TXT record strings
+as a separation is implied by using multiple TXT records.
+
+TXT records MAY be queried before SRV records because for example they could
+include the ``replicaSet`` option to define the replica set name.
+
+A Client MUST use options specified in the Connection String to override
+options provided through TXT records.
```

Comment:
And also the client MUST use the options provided as parameters in code to the MongoClient constructor, or equivalent API for each driver, to override the TXT records.

(The Connection String Spec doesn't specify whether URI or parameters take precedence. But clearly they MUST both override TXT records.)

Reply:
Done